src/main/java/it/pagopa/pn/deliverypush/actions/LegalFactPdfGeneratorUtils.java

```diff
@@ -216,15 +219,12 @@ private TimelineElement timelineElement(Action action) {
         }
 		return row.get();
 	}
-	
+	    
     public String instantToDate(Instant instant) {
-        OffsetDateTime odt = instant.atOffset(ZoneOffset.UTC);
-        int year = odt.get(ChronoField.YEAR_OF_ERA);
-        int month = odt.get(ChronoField.MONTH_OF_YEAR);
-        int day = odt.get(ChronoField.DAY_OF_MONTH);
-		int hour = odt.get(ChronoField.HOUR_OF_DAY);
-		int min = odt.get(ChronoField.MINUTE_OF_HOUR);
-        return String.format("%04d-%02d-%02d %02d:%02d", year, month, day, hour, min);
+    	ZoneId zoneId = ZoneId.of("Europe/Rome");
+    	LocalDateTime localDate = LocalDateTime.ofInstant(instant, zoneId);
+    	
+    	return localDate.format( ITALIAN_DATE_TIME_FORMAT );
```

Comment:
Il codice con il semplice formato dd/MM/yyyy HH:mm lascia l'ambiguità dell'ora di passaggio dall'ora solare all'ora legale. Quando si torna indietro di un ora tale ora viene ripetuta due volte generando un ambiguità. Sarebbe bene aggiungere la timezone CEST o CET ma solo in caso di quell'ora che si ripete due volte.
L'aggiunta della timezone aggiunge pesantezza visuale quasi sempre inutile.
Per questo va aggiunta solo dove genera valore: quell'ora che viene ripetuta due volte.